imod/data/sample_data.py

```diff
@@ -275,3 +275,16 @@ def colleagues_river_data(path: Union[str, Path]):
         x_preserve | y_preserve, riv_bot_da + 0.15
     )
     return riv_ds
+
+
+def tutorial_03(path: Path | str) -> None:
+    """
+    Starting dataset for tutorial 3 in the iMOD Documentation.
+    """
+    path = Path(path)
+    filename = "iMOD-Documentation-tutorial_03.zip"
+    lock = FileLock(REGISTRY.path / f"{filename}.lock")
+    with lock:
```

Comment:
Is the path provided to this method unique? Otherwise there can still be a race condition

Reply:
Added a comment